setup/src/bt_srv.c

```diff
@@ -165,17 +164,33 @@ int bt_srv_init(void)
 
 	settings_load();
 
-	LOG_INF("Advertising...");
-
-	while (1) {
-		k_sleep(500);
-
-		if (!active_conn) {
-			adv_tmp = (adv_bool) ? ad_inet6 : ad_mcumgr;
-			advertise(adv_tmp);
-			adv_bool = !adv_bool;
-		}
-	}
+	LOG_INF("Bluetooth read for advertising...");
 
 	return 0;
 }
+
+/* Alternate advertising if not connected */
+void bt_srv_toggle_adv(void)
```

Comment:
`bt_srv_toggle_advertising`.
I think that the function shouldn't be prefixed the advertising. In the case of `bt_srv` is ok because of the file name.
But the function should have a name that avoids me to read the comments about it to know what `adv` means.